src/celeritas/global/alongstep/detail/AlongStepRZMapFieldMsc.hh

```diff
@@ -0,0 +1,49 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2023 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/global/alongstep/detail/AlongStepRZMapFieldMsc.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "corecel/Types.hh"
+#include "celeritas/em/data/FluctuationData.hh"
+#include "celeritas/em/data/UrbanMscData.hh"
+#include "celeritas/em/msc/UrbanMsc.hh"
+#include "celeritas/field/DormandPrinceStepper.hh"
+#include "celeritas/field/MakeMagFieldPropagator.hh"
+#include "celeritas/field/RZMapField.hh"
+#include "celeritas/field/RZMapFieldData.hh"
+
+#include "AlongStepNeutral.hh"
+#include "MeanELoss.hh"  // IWYU pragma: associated
```

Comment:
We decided to use mean energy loss with the "uniform msc" case to make the comparisons with AdePT fairer. Are energy fluctuations are in use by CMS? If so we should use the higher-fidelity model here.

Reply:
Okay now using FluctELoss instead of MeanELoss.